src/components/shared/Draggable.js

```diff
@@ -2,7 +2,24 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+// @flow
+
 import React, { PureComponent, PropTypes } from 'react';
+import type { Milliseconds } from '../../types/units';
+
+type OnMove = (
+  originalValue: { selectionEnd: Milliseconds, selectionStart: Milliseconds },
+  dx: number,
+  dy: number,
+  isModifying: boolean
+) => void;
```

Comment:
Reading this, I have an idea; please tell me if this is too complicated.

In Draggable, I think we can export OnMove: `export type OnMove`. So in `SelectionScrubberOverlay.js` we could import the type from Draggable: `import type { OnMove } from '../shared/Draggable`.

I think this makes sense because `Draggable` actually defines the type, in `SelectionScrubberOverlay` we only use it.